UI/index.html

```diff
@@ -0,0 +1,554 @@
+<html>
+
+  <head>
+    <meta charset="UTF-8" />
+    <title>Title</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+
+
+  <body>
+
+    <header>
+      <div class="logo">
+        <img
+          width="50px"
+          height="50px"
+          src="https://image.winudf.com/v2/image/Y29tLkNvbG9yU2NyZWVucy5CbGFja0FwcF9zY3JlZW5fMF8xNTM1ODY4NDU4XzA3Mg/screen-0.jpg?h=500&fakeurl=1&type=.jpg"
+        />
+      </div>
+      <div>
+        <span class="title">Title</span>
+        <span class="helloMessage">Здравствуйте, Гость/Username!</span>
+        <form>
+          <button class="authorization">Войти/Выйти</button>
+        </form>
+      </div>
+    </header>
+
+    <button class="addAdvertisement">Добавить объявление</button>
```

Comment:
Since HTML and CSS are case-insensitive, it's not the best idea to use camel case for class names